vulture/core.py

```diff
@@ -521,6 +535,10 @@ def main():
     args = _parse_args()
     vulture = Vulture(verbose=args.verbose)
     vulture.scavenge(args.paths, exclude=args.exclude)
+    if args.make_whitelist:
```

Comment:
Let's use `report_func = vulture.make_whitelist if args.make_whitelist else vulture.report` and then `sys.exit(report_func(...))` to avoid repeating code.